src/app/team-view/team-view.component.scss

```diff
@@ -0,0 +1,55 @@
+img{
+    height: 420px;
```

Comment:
jak masz container to lepiej jakbyś napisał 100% po prostu bo w ten sposób nie będziesz musiał zmieniać w dwóch miejscach gdyby rozmiar się zmienił, zresztą jak tam na dole jest flex ustawiony to prawdopodobnie nie potrzebujesz tutaj ustawiac rozmiaru w ogole ale nie jestem pewien